packages/init/__tests__/logInOrBypass.test.ts

```diff
@@ -37,29 +37,12 @@ describe("loginOrBypass", () => {
   const userId = "1234567";
   const shortId = "12";
   const intercomHash = "intercomHash";
-  const type = "USER";
-  const repositories = {
-    "foo-repo": { dbid: "abcd", role: Models.Roles.OWNER },
-    qwerty: { dbid: "efgh", role: Models.Roles.WRITER },
-  };
 
-  test("is should validate the session and get the users profile", async () => {
+  test("it should validate the session and get the users profile", async () => {
     jest.spyOn(fs, "lstatSync").mockImplementation(() => ({} as fs.Stats));
     jest.spyOn(fs, "writeFileSync");
 
-    jest
-      .spyOn(fs, "readFileSync")
-      .mockReturnValue(JSON.stringify({ cookies: fakeCookie, base: fakeBase }));
-
-    nock("https://auth.prismic.io")
```

Comment:
no, I changed the code to call the profile route which will return a 401 if the user isn't logged in.
I didn't that to avoid having to do a validate session and then call the profile.